data/protocols/1/CurveFinance.json

```diff
@@ -13,7 +13,7 @@
     },
     "es": {
       "name": "Curve Finance",
-      "description": "Curve Finance is an automated market maker protocol designed for token swaps with low fees and slippage. It's a decentralized liquidity aggregator where users can add their assets to several different liquidity pools and earn fees."
+      "description": "Curve Finance es un protocolo de creación de mercado automatizado diseñado para intercambios de tokens con tarifas bajas y deslizamiento. Es un agregador de liquidez descentralizado donde los usuarios pueden agregar sus activos a varios grupos de liquidez diferentes y ganar parte de las tarifas."
```

Comment:
AS-IS: "Curve Finance es un protocolo de creación de mercado automatizado diseñado para intercambios de tokens con tarifas bajas y deslizamiento. Es un agregador de liquidez descentralizado donde los usuarios pueden agregar sus activos a varios grupos de liquidez diferentes y ganar parte de las tarifas."

TO-BE: "Curve Finance es un protocolo de creación de mercado automatizado diseñado para intercambios de tokens con tarifas y deslizamiento bajos. Es un agregador de liquidez descentralizado donde los usuarios pueden agregar sus activos a varios grupos de liquidez diferentes y ganar parte de las tarifas."

Reply:
Done